controlplane/kubeadm/config/default/kustomization.yaml

```diff
@@ -0,0 +1,74 @@
+# Adds namespace to all resources.
+namespace: kubeadm-system
```

Comment:
kubeadmcontrolplane-system or potentially kcp-system?

Reply:
i'll go with short. we have at least until this release to figure out a strategy (or find out this is good enough)